boards/redboard_redv/Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile for building the tock kernel for the HiFive1 platform
+
+TARGET=riscv32imac-unknown-none-elf
+PLATFORM=hifive1
+QEMU ?= qemu-system-riscv32
+
+include ../Makefile.common
+
+# Default target for installing the kernel.
+.PHONY: install
+install: flash
+
+flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
```

Comment:
Doesn't this just flash the hfive1 binary as `PLATFORM` is hifive1?

Reply:
fixed